src/coffee/client.coffee

```diff
@@ -220,9 +220,8 @@ class SphereClient
   #
   # options - An {Object} to configure the client
   constructor: (options = {}) ->
-
     # Private: instance of a {TaskQueue}
-    @_task = options.task or new TaskQueue
+    @_task = options.task or new RepeaterTaskQueue {}, { attempts: 50, timeout: 200, timeoutType: 'v' }
```

Comment:
I do not get what you mean by _Exponentials backoff_ and how it explains the fact that we want to do 50 retries on a request in 200 ms.